src/components/AccountDrawer/UniwalletModal.tsx

```diff
@@ -122,7 +122,7 @@ function InfoSection({ onClose }: { onClose: () => void }) {
         </ThemedText.Caption>
       </AutoColumn>
       <Column>
-        <DownloadButton onClick={onClose} />
```

Comment:
Will H mentioned we should leave the modal open, rather than close it after download is clicked